search/commands_test.py

```diff
@@ -1,54 +0,0 @@
-"""
```

Comment:
moved these test cases to search/api_test.py since it's testing search/api.py functionality